python/src/ngraph/ops.py

```diff
@@ -2588,3 +2588,82 @@ def reverse_sequence(input, seq_lengths, batch_axis, seq_axis, name=None):
     """
     return _get_node_factory().create('ReverseSequence', [input, as_node(seq_lengths)],
                                       {'batch_axis': batch_axis, 'seq_axis': seq_axis})
+
+
+@nameable_op
+def bucketize(data, buckets, output_type='i64', with_right_bound=True):
+    # type: (Node, NodeInput, str, bool) -> Node
+    """Return a node which produces the Bucketize operation.
+
+    :param data:              Input data to bucketize
+    :param buckets:           1-D of sorted unique boundaries for buckets
+    :param output_type:       Output tensor type, "i64" or "i32", defaults to i64
+    :param with_right_bound:  indicates whether bucket includes the right or left
+                              edge of interval. default true = includes right edge
+    :return: Bucketize node
+    """
+    return _get_node_factory().create(
+        'Bucketize',
+        [data, as_node(buckets)],
+        {'output_type': output_type, 'with_right_bound': with_right_bound},
+    )
+
+
+@nameable_op
+def range(start, stop, step):
+    # type: (Node, NodeInput, NodeInput) -> Node
+    """Return a node which produces the Range operation.
+
+    :param start:  The start value of the generated range
+    :param stop:   The stop value of the generated range
+    :param step:   The step value for the generated range
+    :return: Range node
+    """
+    return _get_node_factory().create('Range', [start, as_node(stop), as_node(step)])
+
+
+@nameable_op
+def region_yolo(input,  # type: Node
+                coords,  # type: int
+                classes,  # type: int
+                num,  # type: int
+                mask,  # type: List[int]
+                axis,  # type: int
+                end_axis,  # type: int
+                do_softmax=True,  # type: bool
+                anchors=[],  # type: List[float]
```

Comment:
AFAIK,  In Python we should't set a list `[]` (mutable) as default argument here but init such an arg in the function body.
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments